<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<launch>

    <node name="background_filter" pkg="background_filter" type="background_filter_node" output="screen" ns="bird_vs_drone">
        <rosparam file="$(find background_filter)/config/background_filter.yaml" command="load"/>
    </node>

</launch>